src/sass/components/_forms.scss

```diff
@@ -0,0 +1,54 @@
+.c-trial-form {
+	display: flex;
+	flex-direction: column;
+	margin-top: 2rem;
+	background-color: $white;
+	justify-content: space-between;
+	overflow: hidden;
+	width: 30rem;
+	height: 30rem;
```

Comment:
Why do we need height of form? Elements inside form should create height. Do not add height to the element if you can.